### PR TITLE
Clean up loc strings

### PR DESCRIPTION
**Pull request by jmarolf**
ran some tooling to find duplicate loc strings. There are actually a lot duplication in the codestyle layer that I will follow up on

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
image


I'm curious why the directio here goes from Editor to VS.

File: src/VisualStudio/CSharp/Impl/EditorConfigSettings/DataProvider/Whitespace/CSharpWhitespaceSettingsProvider.cs [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+            yield return WhitespaceSetting.Create(CSharpFormattingOptions2.IndentSwitchSection, CSharpVSResources.Indent_case_labels, editorConfigOptions, visualStudioOptions, updaterService, FileName);
+            yield return WhitespaceSetting.Create(CSharpFormattingOptions2.LabelPositioning, CSharpVSResources.Label_Indentation, editorConfigOptions, visualStudioOptions, updaterService, FileName);
+        }
+
+        private IEnumerable<WhitespaceSetting> GetWrappingOptions(AnalyzerConfigOptions editorConfigOptions, OptionSet visualStudioOptions, OptionUpdater updaterService)
+        {
+            yield return WhitespaceSetting.Create(CSharpFormattingOptions2.WrappingPreserveSingleLine, CSharpVSResources.Leave_block_on_single_line, editorConfigOptions, visualStudioOptions, updaterService, FileName);
+            yield return WhitespaceSetting.Create(CSharpFormattingOptions2.WrappingKeepStatementsOnSingleLine, CSharpVSResources.Leave_statements_and_member_declarations_on_the_same_line, editorConfigOptions, visualStudioOptions, updaterService, FileName);
+        }
+    }
```
Comment (CyrusNajmabadi) [inline src/VisualStudio/CSharp/Impl/EditorConfigSettings/DataProvider/Whitespace/CSharpWhitespaceSettingsProvider.cs:105]: what caused this big add?

**Comment by jmarolf** [comment]
> I'm curious why the directio here goes from Editor to VS.

Because all the strings live in the VS layer. I could work to move all the Tools-> options code down to editor but that does not really make sense to me

File: src/VisualStudio/CSharp/Impl/EditorConfigSettings/DataProvider/Whitespace/CSharpWhitespaceSettingsProvider.cs (hunk 1 above)
Comment (jmarolf) [reply src/VisualStudio/CSharp/Impl/EditorConfigSettings/DataProvider/Whitespace/CSharpWhitespaceSettingsProvider.cs:105]: This isn't an add this is a move. Github just doesn't know how to display moves correctly. I would review this PR one commit at a time

**Review by CyrusNajmabadi — APPROVED** [review]
LGTM.  Thanks!

**Pull request merged**